api.md

```diff
@@ -1832,31 +1832,68 @@ DROP MATERIALIZED VIEW contagg_view;
 
 ## refresh_continuous_aggregate() :community_function: [](refresh_continuous_aggregate)
 
-Refresh all buckets of a continuous aggregate between two points of
+Refresh all buckets of a continuous aggregate between two points in
 time. 
 
-The function expects the parameter values to have the same time type
-as used in the continuous aggregate's time bucket expression (e.g., if
-the time bucket specifies in `timestamptz`, then the start and end time
-supplied should also be `timestamptz`).
+The refresh will only materialize at the granularity of a bucket, as
+determined by the continuous aggregate view query's `time_bucket`
+expression. Therefore, only buckets that are completely covered by the
+_refresh window_ will be updated, and the window should account for
+any time zone shifts when applicable (note that a continuous aggregate
+using the `DATE` or `TIMESTAMP WITH TIME ZONE` type aligns with the
+UTC time zone).
+
+The function expects the window parameter values to have the same time
+type as used in the continuous aggregate's time bucket expression
+(e.g., if the time bucket specifies in `TIMESTAMP WITH TIME ZONE`,
+then the start and end time supplied should also be `TIMESTAMP WITH
+TIME ZONE`).
+
 
 #### Required Arguments [](refresh_continuous_aggregate-required-arguments)
 
 |Name|Description|
 |---|---|
 | `continuous_aggregate` | (REGCLASS) The continuous aggregate to refresh. |
-| `window_start` | Start of the window to refresh, has to be before `window_end`. `NULL` is eqivalent to `MIN(timestamp)` of the hypertable. |
-| `window_end` | End of the window to refresh, has to be after `window_start`. `NULL` is eqivalent to `MAX(timestamp)` of the hypertable. |
+| `window_start` | Start of the window to refresh (inclusive), has to be before `window_end`. `NULL` is equivalent to refreshing from the beginning of time. |
```

Comment:
```suggestion
| `window_start` | Start of the window to refresh (inclusive), has to be before `window_end`. `NULL` is equivalent to refreshing from the earliest point of data in the underlying hypertable |
```

Reply:
Technically, this isn't correct. Let's say you removed/dropped data in the hypertable. You are still able to refresh those regions that have no data and actually remove the data in the continuous aggregate.